elastic/kibana
# [Discover] Unskip Discover-Lens functional test suite

## Summary

Fixes #184600

The PR catches propagating an invalid state of properties to the Lens embeddable, that causes this error, when e.g. ad hoc data views are being edited causing the following error when rendering the histogram in Discover:

![image](https://github.com/user-attachments/assets/bd689168-8362-4089-9020-0772ae9050d1)


Note: this midigates and catches an invalid state, that we ideally should catch before it happens, but the current approach fixes the problem, but we should to refactor this to stabilize and ideally simplify.

### Testing

There was a reliable way to run into this error before this PR:

* Install all sample data.
* On Discover page, create an ad hoc data view kib* with **@timetamp** as the time field. 
* Edit data view and set timestamp as the time field. 
* Edit again and remove the time field. 
* Edit again and set **timestamp** as the time field.

This caused the histogram to fail, but this should no longer be the case


### Checklist
